nncf/quantization/algorithms/definitions.py

```diff
@@ -22,3 +22,27 @@ class Granularity(Enum):
 class RangeType(Enum):
     MINMAX = 'min_max'
     MEAN_MINMAX = 'mean_min_max'
+
+
+class OverflowFix(Enum):
+    """
+    This option controls whether to apply the overflow issue fix for the 8-bit quantization.
+
+    8-bit instructions of older Intel CPU generations (based on SSE, AVX-2, and AVX-512 instruction sets)
+        suffer from the so-called saturation (overflow) issue: in some configurations,
+        the output does not fit into an intermediate buffer and has to be clamped.
+    This can lead to an accuracy drop on the aforementioned architectures.
+    The fix set to use only half a quantization range to avoid overflow for specific operations.
```

Comment:
```suggestion
    8-bit instructions of older Intel CPU generations (based on SSE, AVX-2, and AVX-512 instruction sets)
    suffer from the so-called saturation (overflow) issue: in some configurations,
    the output does not fit into an intermediate buffer and has to be clamped.
    This can lead to an accuracy drop on the aforementioned architectures.
    The fix set to use only half a quantization range to avoid overflow for specific operations.
```

Reply:
Done